src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
+
+html{
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Good job on using a fallback font. In case the `Roboto` font fails to load, the browser will use `Arial`. If `Arial` is not available, it will use any available `sans-serif` font.